xbmc/music/MusicDatabase.cpp

```diff
@@ -502,7 +502,17 @@ bool CMusicDatabase::AddAlbum(CAlbum& album, int idSource)
       PrepareSQL("UPDATE album SET iDisctotal = (SELECT COUNT(DISTINCT iTrack >> 16) FROM song "
                  "WHERE song.idAlbum = album.idAlbum) WHERE idAlbum = %i",
                  album.idAlbum));
-
```

Comment:
// if has at least 3 distinct disc subtitles then make album a boxset
or something like that